client/lib/posts/test/utils.js

```diff
@@ -67,6 +67,30 @@ describe( 'utils', function() {
 		} );
 	} );
 
+	describe( '#isBackDatePublished', function() {
+		it( 'should return false when no post is supplied', function() {
+			assert( ! postUtils.isBackDatedPublished() );
+		} );
+
+		it( 'should return false when status !== future', function() {
+			assert( ! postUtils.isBackDatedPublished( { status: 'draft' } ) );
+		} );
+
+		it( 'should return false when status === future and date is in future', function() {
+			const tenMinutes = 1000 * 60;
+			const postDate = +new Date() + tenMinutes;
```

Comment:
`Date.now()` is a more readable variant of `+new Date()`, latter seen usually when pre-ES5 support needed, IE<9.